js/modules/render/loader.mjs

```diff
@@ -0,0 +1,12 @@
+
+/**
+ * Adds loader to container
+ * @param {Element} container 
+ */
+export function addLoader(container){
+  container.innerHTML = `<div class="loader">
+                          <div class="outer-loader"></div>
+                          <div class="inner-loader"></div>
+                          <p>Getting products, please wait...</p>
```

Comment:
This loader states we are getting products; however, there aren't products on the page.

Reply:
Loader now says "Getting posts, please wait..."